GcmEndpoints/README.md

```diff
@@ -76,26 +58,14 @@ When you added this backend module to your project, the [required permissions, n
 
 Furthermore, a `RegistrationEndpoint` [Cloud Endpoints API](https://developers.google.com/appengine/docs/java/endpoints/) has been automatically generated for you, so that you could start calling this endpoint from your Android app to register devices with your new Google Cloud Messaging backend.
 
-Here is an example code snippet which illustrates how to create an [AsyncTask](http://developer.android.com/reference/android/os/AsyncTask.html) to register the user's device with your new backend:
-
-```java
-class GcmRegistrationAsyncTask extends AsyncTask<Void, Void, String> {
-    private static Registration regService = null;
-    private GoogleCloudMessaging gcm;
-    private Context context;
+Set up your [Google Cloud Messaging Client](https://developers.google.com/cloud-messaging/android/client) in order to obtain a registration token, register the user's device with your new backend, and to set up the listener services to show push notifications from the GCM backend.
 
-    // TODO: change to your own sender ID to Google Developers Console project number, as per instructions above
-    private static final String SENDER_ID = "1234567890123";
+Here is an example code snippet to send your newly obtained GCM registration token to the server which you should invoke from the `RegistrationIntentService` you created when setting up the client:
```

Comment:
I'm having trouble following this sentence. I suggest breaking it into two. Also, "Here is an example code snipped to send" --> "This code snippet sends"